SConstruct

```diff
@@ -1312,6 +1312,10 @@ if doConfigure :
 		sys.stderr.write( "ERROR : unable to find the Blosc libraries - check BLOSC_INCLUDE_PATH and BLOSC_LIB_PATH.\n" )
 		Exit( 1 )
 
+	if not c.CheckLibWithHeader( env.subst( "boost_wave" + env["BOOST_LIB_SUFFIX"] ), "boost/wave.hpp", "CXX" ):
+		sys.stderr.write( "WARNING : unable to find boost wave, disabling IECoreGL - check BOOST_LIB_PATH.\n" )
+		env["WITH_GL"] = False
```

Comment:
I think we can do better than this by moving this check just below the check for GLEW, using the `Configure` object dedicated to IECoreGL. Before we were needlessly linking `boost_wave` into IECore and all the other libs. Moving it will mean we only link it into IECoreGL.